packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -19,6 +19,17 @@ const _ = require('lodash')
 
 const rhosts = /github|bitbucket|gitlab/i
 
+// TODO: delete when min node target is set to 12.10+
+Promise.allSettled = Promise.allSettled || function (promises) {
+  const wrappedPromises = promises.map((p) =>
+    Promise.resolve(p).then(
+      (value) => ({ status: 'fulfilled', value }),
+      (reason) => ({ status: 'rejected', reason })
+    )
+  )
+  return Promise.all(wrappedPromises)
+}
```

Comment:
will try to use native function if available, else apply polyfill

https://github.com/kriskowal/q/blob/01252cbe917178f5935cbe2a1ab43e12934c7057/q.js#L1693-L1703


https://node.green/#ES2020-features--Promise-allSettled